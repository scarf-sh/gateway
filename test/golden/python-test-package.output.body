<!DOCTYPE html><html><body><h1>numpy</h1>
<a href="/simple/numpy/numpy-1.3.0.tar.gz">numpy-1.3.0.tar.gz</a><br>
<a href="/simple/numpy/numpy-1.4.0.tar.gz">numpy-1.4.0.tar.gz</a><br>
</body></html>