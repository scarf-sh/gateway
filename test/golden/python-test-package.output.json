{
    "capture": "Just (PythonCapture {pythonCapturePackage = Just \"123\", pythonCaptureVersion = Nothing, pythonCaptureFileName = Nothing, pythonCaptureFileBackendURL = Nothing})",
    "headers": {
        "Content-Length": "196",
        "Content-Type": "text/html",
        "Etag": "\"nKXJKw6bVHghW5Egg1iiVvuSBLklJQSE0o9KKqNJllU=\""
    },
    "status": 200
}